src/primitives/block.h

```diff
@@ -25,7 +39,7 @@ class CBlockHeader
 public:
     static const int32_t CURRENT_VERSION=SIZE_FORK_VERSION;
     // This code knows the rules for these block versions:
-    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 3 | 2 | 1);
+    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 4 | 3 | 2 | 1);
```

Comment:
I guess that before the versions weren't used as bits since 3 is not a bit. If the point is to make clear that it supports BIP66 and BIP65 and what not, it would help to define constants for those and use those.

static const int32_t BIP66 = 3;
static const int32_t BIP65 = 4;
...
static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | BIP65 | BIP66 | ...);